### ISSUE TITLE
Support multi language and multi webspace

### ISSUE THREAD

**Issue by QuingKhaos** [sulu-io/ExampleNewsBundle#19]
May you do another blog post, where you show how to integrate support for multi language and multi webspace? This would help to do bundles for complex customers and also publish them as open source because they are usable in every installation.

**Comment by sjoerdmaessen** [comment]
Yes, this would be very helpful, especially since I could see myself copying this bundle for different purposes and projects. I really like the current blogpost series to get an understanding of the system.

**Comment by wachterjohannes** [comment]
@patkar @sjoerdmaessen i hope i will have time soon but i also see the need to do this! if you have any other ideas how we could extend the blogs create further issues (:
